content/git/concepts/pull/pull.md

```diff
@@ -41,19 +41,19 @@ Prerequisites to pulling from a remote repository include:
   nothing to commit, working tree clean
   ```
 
-- Having a valid remote repository set, in this case there is remote repository called `companyname-dev`:
+- Having a valid remote repository set. In this case the remote repository is named `origin` and can be found at the url `https://github.com/CompanyName/product-dev`:
 
   ```shell
   $ git remote -v
 
-  companyname-dev    https://github.com/CompanyName/product-dev.git (fetch)
-  companyname-dev    https://github.com/CompanyName/product-dev.git (push)
+  origin    https://github.com/CompanyName/product-dev.git (fetch)
+  origin    https://github.com/CompanyName/product-dev.git (push)
   ```
 
 At this stage, pulling is as simple as using the basic syntax with the correct remote and branch names:
```

Comment:
```suggestion
At this stage, pulling can accomplished with the names of a valid remote and branch:
```

Reply:
Edits:
- can be accomplished*
- "the names of a valid remote and branch:" -> "a valid remote and branch name:"